### PR TITLE
Adding support for java layer

### PR DESCRIPTION
**Pull request by katiewest820**
- Adding support for java layer
- Allow user to be able to chose between more than one layer version
https://newrelic.atlassian.net/browse/LAMBDA-1226

### CODE REVIEW

**Review by mrickard — APPROVED** [review]

File: newrelic_lambda_cli/layers.py [hunk 1]
```diff
@@ -69,10 +95,10 @@ def _add_new_relic(input, config, nr_license_key):
         if not layer["Arn"].startswith(utils.get_arn_prefix(aws_region))
     ]
 
-    new_relic_layers = []
+    new_relic_layer = []
```
Comment (kolanos) [inline newrelic_lambda_cli/layers.py:98]: Why this change?

Comment (katiewest820) [reply newrelic_lambda_cli/layers.py:98]: because there is only ever one layer in the list. I thought it was clearer.

**Review by kolanos — APPROVED** [review]
